pkg/knuu/instance.go

```diff
@@ -258,11 +258,18 @@ func (i *Instance) AddFile(src string, dest string, chown string) error {
 	return nil
 }
 
+// AddFile adds a file to the instance
+// This function can only be called in the state 'Preparing'
+func (i *Instance) AddFile(src string, dest string, chown string) error {
+	// As dockers `ADD` works for both files and folders, we can just call Add here
+	return i.AddFile(src, dest, chown)
+}
+
 // AddFolder adds a folder to the instance
 // This function can only be called in the state 'Preparing'
 func (i *Instance) AddFolder(src string, dest string, chown string) error {
-	// As dockers `ADD` works for both files and folders, we can just call AddFile here
-	return i.AddFile(src, dest, chown)
+	// As dockers `ADD` works for both files and folders, we can just call Add here
+	return i.Add(src, dest, chown)
```

Comment:
if they do the same because of docker, why keep both functions?, we could have only `Add` and do it there, wdyt?

Reply:
I like to keep the separate functions as the logic of adding folders and files might change. I already thought of checking whether the files the developer wants to add exist. This might differ between folders and files, idk.